src/server/services/persist-upload.js

```diff
@@ -26,7 +26,7 @@ async function extractDocuments (buffer) {
   for (const sheetName of workbook.SheetNames) {
     documents.push({
       type: normalizeSheetName(sheetName),
-      content: xlsx.utils.sheet_to_json(workbook.Sheets[sheetName], { header: 1 })
+      content: xlsx.utils.sheet_to_json(workbook.Sheets[sheetName], { header: 1, range: 'A1:AU200' })
```

Comment:
i set this to avoid the problem discussed in slack, of having >1M rows in my spreadsheets. it might be nice to continue leaving this here? maybe we can limit it to something more sensible, like, say, 20k rows?

slack link: https://usdigitalresponse.slack.com/archives/C031R1Y49KL/p1651169022588859

Reply:
oh, interesting. i haven't tried that, i was assuming it would still be slow because it will attempt to iterate over the 1M rows to see if they're empty. the resulting data structure from this call does not have 1M json rows, so i think it's the attempt-to-access that's slow, but i'll double-check.

Reply:
Oh, I see your point.  I forgot the read performance itself was the bottleneck.  Worth testing still, I guess.

Reply:
yeah, still a problem with `blankrows: false` --

```
2022-05-12T15:45:42.606Z persist-uploads extracting Certification (A1:D4
[dev:*express] 2022-05-12T15:45:42.606Z persist-uploads extracting Cover (A1:I13
[dev:*express] 2022-05-12T15:45:42.607Z persist-uploads extracting EC 1 - Public Health (A1:AR1048576
[dev:*express] 2022-05-12T15:46:14.961Z persist-uploads extracting EC 2 - Negative Economic Impact (A1:AU200
[dev:*express] 2022-05-12T15:46:14.966Z persist-uploads extracting EC 3 - Public Sector Capacity (A1:AJ200
```

ec1 ref is to 1048576 rows, and it takes 30 seconds